intialised clear_color variable to make the code more consistent

freshBakedPie314 opened the pull request:
initialised clear_code variable to p_default_bg_color during declaration to makw it consistent with it's mobile renderer version counterpart render_forward_mobile.cpp

<!--
Please target the `master` branch in priority.
PRs can target `3.x` if the same change was done in `master`, or is not relevant there.

Relevant fixes are cherry-picked for stable branches as needed by maintainers.
You can mention in the description if the change is compatible with `3.x`.

To speed up the contribution process and avoid CI errors, please set up pre-commit hooks locally:
https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html
-->

AThousandShips commented:
This isn't needed, it is set in all code paths so this assignment has no effect

freshBakedPie314 committed:
intialised clear_color variable to make the code more consistent

initialised clear_code variable to p_default_bg_color during declaration to makw it consistent with it's mobile renderer version counterpart render_forward_mobile.cpp
